ephios/core/signup/stats.py

```diff
@@ -34,6 +67,28 @@ def replace(self, **kwargs):
     def has_free(self):
         return self.free is None or self.free > 0
 
+    @property
+    def full_count(self):
+        """
+        Returns the maximum number of participants so the object is considered full.
+        This can be different from `max_count`, if e.g. someone dispatched an
+        unqualified volunteer, so there is still an open spot.
+        """
+        if self.max_count is None or self.free is None:
+            return None
+        return max(
+            self.max_count,
+            self.confirmed_count + self.free,
+        )
```

Comment:
Is there actually a case where `confirmed_count + free < max_count`? I can't think of an example.